### PR TITLE
Update Home and About Page

### PR DESCRIPTION
**Pull request by jpvelez**
This PR does 3 things:

1. Adds a bunch of copy to the about page
2. Removes things from the home page, leaving only the title and map
3. Renames the site to Decarb My State

I need help getting the charts I removed from the home page onto the about page.

### CODE REVIEW

**Review by vkoves — COMMENTED** [review]
Have one quick comment, but which graphs did you want to move over from the homepage @jpvelez? The line graph of emissions and the clean electrification graph, over just the latter?

File: src/pages/about.js [hunk 1]
```diff
@@ -1,42 +1,101 @@
...
+          <p>Nuclear is <strong>safe</strong> and <strong>reliable</strong>. Though expensive, building more nuclear plants would allow us to build fewer wind and solar farms.</p> 
+          <p>That said, this site <em>intentionally</em> depicts how much wind and solar we need to decarbonize every state, and argues that this "all-renewables" approach is fact feasible (and affordable) using <strong>overbuilding</strong>.</p>
+
           <h2 className="pt-3">Data and sources</h2>
           <p>All of the data used on this website are from publicly available and trusted government and nonprofit sources. Below is a description of the primary datasets and their publishers. For more details on how we discovered and worked with this data, take a look at our <a href="https://github.com/chihacknight/decarbonize-my-state/issues?q=label%3Adata">GitHub issue tracker for data</a> and our <a href="https://github.com/chihacknight/decarbonize-my-state/tree/main/data">data folder</a> of this open source project.</p>
           
-          <h3 className="pt-2">US Emissions by State</h3>
+          <h4 className="pt-2">US Emissions by State</h4>
```
[the remark this answers is not in the document]
Comment (jpvelez) [reply src/pages/about.js:91]: Just to make each dataset section look smaller, so the hierarchy is clearer. Right now the font-size for the various headers are close enough to each other that most people won't distinguish the nesting.

[the remark this answers is not in the document]
Comment (vkoves) [reply src/pages/about.js:91]: For accessibility you shouldn't skip levels like that, I'd just tweak the associated CSS class then

**Review by derekeder — APPROVED** [review]
i took a pass on this and made a few changes:

- resolved the `h3` / `h4` issue @vkoves raised
- added Surag and Robert to the credits
- small text changes adding clarity on the about page
- changed the link color to the same dark brown we're using in the maps
- resolved the merge conflict with `main`

I also set up https://decarbmystate.com. The decarbonizemystate.com URL is now set up as a redirect.

@vkoves I'm good to merge this in if you are

**Comment by vkoves** [comment]
@derekeder - didn't we want to add in the old American emissions graph or at least the clean electrification graph from the old homepage to the new about page? I'm also fine making a separate issue for that, but I have a feeling you'd be able to knock that out really easily

**Comment by derekeder** [comment]
@vkoves ah yes - I can add that back in

**Comment by derekeder** [comment]
@vkoves chart re-added
Screen Shot 2022-07-05 at 9 02 06 PM

**Review by vkoves — APPROVED** [review]
Looks good to me! Some of the spacing of the About page feels a little clunky and tight, but that's something we can tune later.

**Pull request merged**